yt_dlp/extractor/boosty.py

```diff
@@ -202,6 +215,8 @@ def _real_extract(self, url):
                         'thumbnail': (('previewUrl', 'defaultPreview'), {url_or_none}),
                     }, get_all=False)})
 
+        if not post.get('hasAccess'):
+            self.raise_login_required('This post requires a subscription', True, method='cookies')
         if not entries:
```

Comment:
I'd also check for entries here in case the site stops using the `hasAccess` key. Can remove `method='cookies'`, since that's the default for extractors that don't set `_NETRC_MACHINE` 
```suggestion
        if not entries and not post.get('hasAccess'):
            self.raise_login_required('This post requires a subscription', metadata_available=True)
        elif not entries:
```

Reply:
Makes sense. Why did you remove the `method=cookies`? I tried that and it still prints the same message about using "--cookies-from-browser or --cookies" so it doesn't really matter, I thought I'd explicitly add that since only cookie authentication works (I don't know what other methods there are).

Reply:
Huh I looked at the code and forgot to read the actual message. I'll remove `method=cookies` too.